src/main/kotlin/no/nav/syfo/service/SendVarselService.kt

```diff
@@ -40,13 +49,15 @@ class SendVarselService(
             val userAccessStatus = accessControlService.getUserAccessStatus(pPlanlagtVarsel.fnr)
             val fnr = userAccessStatus.fnr!!
             val uuid = pPlanlagtVarsel.uuid
+            val birthDate = pdlConsumer.hentPerson(fnr)?.getFodselsdato()?.let { parsePDLDate(it) }
+            log.info("[FODSELSDATO] Parsed fodselsdato: ${birthDate}")
 
             val varselUrl = varselUrlFromType(pPlanlagtVarsel.type)
             val varselContent = varselContentFromType(pPlanlagtVarsel.type)
             val orgnummer = pPlanlagtVarsel.orgnummer
 
             if (varselUrl !== null && varselContent !== null) {
-                if (userSkalVarsles(pPlanlagtVarsel.type, userAccessStatus)) {
+                if (userSkalVarsles(pPlanlagtVarsel.type, userAccessStatus, birthDate)) {
```

Comment:
Jeg tenker at det er mer fornuftig at `MerVeiledningVarselFinder` filtrerer bort de som er over 67 år. Da samler vi all beregning av hvem som skal ha 39ukersvarsel på ett sted. Da slipper vi også å kalle PDL for de som skal ha aktivitetskravvarsel.

Reply:
Uavhengig av hvor sjekken ligger (ref kommentar over), så er jeg litt bekymret for at denne sjekken vil gjøre jobben ytterligere treig og ustabil 😨

Reply:
Vi får bare kjøre på og se hvor lang tid det tar i prod tenker jeg. Det vil jo kanskje være noen færre varsler som sendes, så kanskje totalen ikke er så forskjellig.

Reply:
> Vi får bare kjøre på og se hvor lang tid det tar i prod tenker jeg. Det vil jo kanskje være noen færre varsler som sendes, så kanskje totalen ikke er så forskjellig. Men det kan kanskje være greit å legge inn loggmelding så vi ser hvor lang tid 67-årssjekken tar også, sånn som vi gjorde med sykmeldinger

Enig i det!

Reply:
> Jeg tenker at det er mer fornuftig at `MerVeiledningVarselFinder` filtrerer bort de som er over 67 år. Da samler vi all beregning av hvem som skal ha 39ukersvarsel på ett sted. Da slipper vi også å kalle PDL for de som skal ha aktivitetskravvarsel.

Enig

Reply:
> are kjøre på og se hvor lang tid det tar i prod tenker jeg. Det vil jo kanskje være noen færre varsler som sendes, så kanskje totalen ikke er så forskjellig. Men det kan kanskje være greit å legge inn loggmelding så vi ser hvor lang tid 67-årssjekken tar og
\\\
Vi kan spøre etter fødselsdato helt i starten når vi lagrer event, og ved usending spøre kun for de brukere som ikke hadde det lagret hos oss